src/overlays/actors/ovl_En_Elforg/z_en_elforg.c

```diff
@@ -607,7 +607,7 @@ void EnElforg_Draw(Actor* thisx, GlobalContext* globalCtx) {
             AnimatedMat_Draw(globalCtx, Lib_SegmentedToVirtual(&D_0402C818));
             break;
     }
-    Matrix_InsertMatrix(&globalCtx->mf_187FC, 1);
+    Matrix_InsertMatrix(&globalCtx->billboardMtxF, 1);
```

Comment:
I know it wasn't you, but could you fix this too? c:
```suggestion
    Matrix_InsertMatrix(&globalCtx->billboardMtxF, MTXMODE_APPLY);
```

Reply:
I double checked and this is already caught in this PR:
https://github.com/zeldaret/mm/pull/454/files